docs/pages/dsHDMIIn/ds-Hdmi-In_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,216 @@
+# Device Settings HdmiIn L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings HdmiIn module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
+- `SoC` \- System on a Chip
+- `HDMI`\- High-Definition Multimedia Interface
+- `HDR` \- High Dynamic Range
+- `HLG` \- Hybrid Log-Gamma
+- `SDR` \- Standard Dynamic Range
+
+### References
+
+- `High Level Test Specification` - [dsHdmiIn High Level TestSpec](ds-hdmi-in_L3_High-Level_TestSpecification.md)
+- `Interface header` - [dsHdmiIn HAL header 4.0.0](https://github.com/rdkcentral/rdk-halif-device_settings/blob/4.0.0/include/dsHdmiIn.h)
+
+## Level 3 Test Cases High Level Overview
+
+|#|Streams Name|Streams description|
+|-|------------|-------------------|
+|1|vts_HDR10_stream|Format: HDR10,Resolution: 3840 x 2160 (4K UHD),Color Depth: 10-bit,Color Space: Rec. 2020|
+|2|vts_SDR_stream|Format: SDR,Resolution: 1920 x 1080|
+|3|vts_HLG_stream|Format: HLG,Resolution: 3840 x 2160 or It can also be used with 1080p and 720p resolutions.|
+|4|vts_DolbyVision_stream|Format: Dolby Vision,Resolution: 3840 x 2160 (4K UHD),Color Depth: 10/12-bit,Color Space: Rec. 2020format and dynamic metadata.|
+|5|vts_HDR10plus_stream|Format: HDR10,Resolution: 3840 x 2160 (4K UHD),Color Depth: 10-bit,Color Space: Rec. 2020 and dynamic metadata capabilities.|
+
+Each test case need to verify with the each HdmiIn port.
+Below are top test use-case for the HdmiIn port.
+
+|#|Test-case|Description|Focus APIs|Source|Sink|
+|-|---------|-----------|----------|------|----|
```

Comment:
Stream numbers is missing in the table. Refer dsAudio and dsVideoport

Reply:
this is one of the comment from gerald to remove streams column , please refer (https://github.com/rdkcentral/rdk-halif-test-device_settings/pull/172) old PR.

Reply:
Here is the comment from gerald
[Ulrond](https://github.com/Ulrond) [yesterday](https://github.com/rdkcentral/rdk-halif-test-device_settings/pull/172#discussion_r1766553414)
if your going to use all streams for all test, then just remove the column, it's not providing value... Since your mentioning streams in the description.